server/courselist.rb

```diff
@@ -458,8 +458,9 @@ def allow_uniqname_override user
     def uniqnameOverride
 
       config_hash = settings.latte_config
+      
       # See if there is a candidate to use as authenticated userid name.
-      uniqname = html_escape(params['UNIQNAME'])
+      uniqname = CourseList.safe_html_escape(params['UNIQNAME'])
```

Comment:
The code is inside "courselist.rb" itself. Do we still need to prefix it with CourseList?